packages/common/component/MetaBindVariable.vue

```diff
@@ -6,6 +6,7 @@
   </slot>
 
   <tiny-dialog-box
+    v-if="dialogShouldInitialize"
```

Comment:
Nice, lazyload is ok.

BTW: adding props `:destroy-on-close="true"` is also work.

Reply:
Got it, thank you. I think using lazyload is fine for now. If in actual scenarios the frequency of opening dialog is not high, switching to destroy-on-close might be better.